frame/clients/maximizing_gtk_app.py

```diff
@@ -0,0 +1,22 @@
+import gi
+try:
+    gi.require_version("Gtk", "4.0")
+except ValueError:
+    gi.require_version("Gtk", "3.0")
```

Comment:
```suggestion
gi.require_version("Gtk", "4.0")
```

Yeah installing `libgtk-4-dev` seems to have worked.